### DISCUSSION TITLE
[Dropdown] Show multiple selection on hover

### DISCUSSION THREAD

**Discussion by unarist** [fomantic/Fomantic-UI#656] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
# Feature Request

## Description

My application mainly uses `on: 'hover'` for dropdowns and I want to use it also with `ui multiple selection dropdown`. However, looks like it doesn't support for now.

So it would be nice if multiple selection dropdown also accepts `on: 'hover'`.

## Example

https://jsfiddle.net/bjucdry4/

![test](https://user-images.githubusercontent.com/705555/55885795-85af3000-5be5-11e9-8c23-ae06982d800f.gif)

**Comment by lubber-de** [comment]
🤔 As long as you don't leave the dropdown list, you can still select as many entries as you like, so for me it works as expected/desired. (opens on hover, closes when leaving list)
Could you please get more precise on what you want to achieve?

**Comment by lubber-de** [comment]
Sorry, my bad 😞 , i should have taken a more deeper look to your provided fiddle (you used mouseenter/leave)..which is a good workaround 🙂

**Comment by lubber-de** [comment]
🤔 It's me again...'hover' seems to actually work as expected with multiple selection
See https://jsfiddle.net/fjvg496d/

**Comment by unarist** [comment, marked as answer]
Ah, you're right. I've confused with `multiple search selection` dropdown, but it would be complicated...
https://jsfiddle.net/ud49gokr/

I should take `multiple selection` on hover or `multiple search selection` on click maybe.

**Comment by lubber-de** [comment]
Yes. That's what I would also suggest. Keyboard entries together with hovering isn't good ui usage. If you don't mind, I would tag this as won't fix and close the issue
